homeassistant/components/comelit/switch.py

```diff
@@ -26,12 +26,16 @@ async def async_setup_entry(
     coordinator: ComelitSerialBridge = hass.data[DOMAIN][config_entry.entry_id]
 
     # Use config_entry.entry_id as base for unique_id because no serial number or mac is available
```

Comment:
Last remark: this comment isn't really commenting anything here.

Reply:
I was asked to add it when added first platform, light.
The idea was to avoid raising questions during following reviews.

Reply:
Yeye, but it's not near the code where it's explaining something